tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -173,6 +228,10 @@ def test_write(self):
             },
         )
 
+        # Transfer the topic to another node
+        self._transfer_topic_leadership()
+
+        # After leadership transfer has completed assert that manifest is OK
```

Comment:
I think this comment might make more sense further down the test (or may even be removed as it looks like we are not checking the manifest structure in this test).